study-builder/tenants/circadia/pages/login.html

```diff
@@ -0,0 +1,546 @@
+
+    <!--[if lte IE 9]>
+      <script src="https://cdn.auth0.com/js/base64.js"></script>
+      <script src="https://cdn.auth0.com/js/es5-shim.min.js"></script>
+    <![endif]-->
+
+    <script src="https://cdn.auth0.com/js/lock/11.30/lock.min.js"></script>
+
+    <style type="text/css">
+      body {
+        font-family: 'Nunito', sans-serif;
+      }
+
+      .auth0-lock {
+        font-family: 'Nunito', sans-serif !important;
+      }
+
+      .auth0-lock.auth0-lock .auth0-lock-overlay {
+        background: radial-gradient(#ffffff, #f3f3f2, #f3f3f2);
+        background-image: radial-gradient(#ffffff, #f3f3f2, #f3f3f2);
+        background-color: #fff;
+      }
+
+      .auth0-lock.auth0-lock.auth0-lock-opened .auth0-lock-widget {
+        box-shadow: 0 0 40px 4px #888888;
+      }
+      .auth0-lock-name {
+        font-size: 12pt !important;
+        text-align: left;
+        padding-left: 0 !important;
+      }
+
+      .auth0-lock-content {
+        padding-top: 10px !important;
+      }
+
+      .auth0-lock-header {
+        display: flex;
+        align-items: center;
+      }
+
+      .auth0-lock-header .auth0-lock-header-bg {
+        background-color: white !important;
+      }
+
+      .auth0-lock.auth0-lock .auth0-lock-form p {
+        text-align: left;
+      }
+
+      .auth0-lock-custom-icon {
+        width: 14px;
+        height: 14px;
+      }
+
+      .auth0-lock-name {
+        padding: 0 !important;
+
+        text-align: center;
+      }
+
+      a:link {
+        text-decoration: underline !important;
+      }
+
+      #ddp-header {
+        margin-top: 10px;
+      }
+
+      #ddp-header .auth0-lock-header-logo {
+        max-width: 100%;
+        height: 60px;
+        margin-bottom: 10px;
+      }
+    </style>
+
+    <script>
+      // Decode utf8 characters properly
+      var config = JSON.parse(
+        decodeURIComponent(escape(window.atob('@@config@@'))),
+      );
+
+      config.extraParams = config.extraParams || {};
+
+      var inviteKey = 'invitationId';
+
+      // save the invitation in local storage to minimize breached password loss-of-invitation
+      if (config.extraParams.invitation_id) {
+        localStorage.setItem(inviteKey, config.extraParams.invitation_id);
+
+        console.log('Saving invitation ' + config.extraParams.invitation_id);
+      } else {
+        var invitation = localStorage.getItem(inviteKey);
+
+        if (invitation) {
+          // unsafe, but better than losing this value during breached password shield
+          config.internalOptions.invitation_id = invitation;
+
+          console.log(
+            'Using stored invitation ' + invitation + ' from local storage',
+          );
+        }
+      }
+
+      var mode = config.extraParams.mode;
+      var study_guid = config.extraParams.study_guid;
+      var headerLogo = '##ASSETS_URL##/auth0/logo.png';
+      var headerText = '';
+      var studyColor = '#0b1245';
+      var studyHelpEmail = '##STUDY_EMAIL##';
```

Comment:
I don't see `STUDY_EMAIL` being provided in `AUTH0_KEYWORD_REPLACE_MAPPINGS` in `circadia-config.json.ctmpl`.